std/algorithm/iteration.d

```diff
@@ -916,12 +923,20 @@ template each(alias pred = "a")
         debug(each) pragma(msg, "Using foreach for ", Iterable.stringof);
         static if (isForeachUnaryIterable!Iterable)
         {
+            debug(each) pragma(msg, "unary");
             foreach (ref e; r)
+            {
                 cast(void)unaryFun!pred(e);
+            }
```

Comment:
Nit/personal preference: curly braces are not needed for simple singe statement blocks.